packages/@ember/object/evented.ts

```diff
@@ -123,19 +76,13 @@ const Evented = Mixin.create({
     This function takes an optional 2nd argument that will become the "this"
     value for the callback. When the target parameter is used the callback method
     becomes the third argument.
-
-    @method one
-    @param {String} name The name of the event
-    @param {Object} [target] The "this" binding for the callback
-    @param {Function|String} method A function or the name of a function to be called on `target`
-    @return this
-    @public
```

Comment:
Ah, linting caught something important for us. All of these annotations need to be copied over to the new form of the docs! They are used by the doc generator!

Reply:
Haha no problem, will do!

Reply:
Done! Please re-review when convenient